### ISSUE TITLE
[Bug]: Collection with BIN_IVF_FLAT index load failed with timeout error

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#30165]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20240121-c8a78465
- Deployment mode(standalone or cluster):cluster
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

```
[2024-01-21T10:18:11.056Z] <name>: deploy_test_index_type_BIN_IVF_FLAT_is_compacted_not_compacted_segment_status_all_is_string_indexed_is_string_indexed_replica_number_1_is_deleted_is_deleted_data_size_3000

[2024-01-21T10:18:11.056Z] <description>: 

[2024-01-21T10:18:11.056Z] <schema>: {'auto_id': False, 'description': '', 'fields': [{'name': 'int64',......  (api_request.py:37)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:15:45 - INFO - ci_test]: inserted 3000 data into collection deploy_test_index_type_BIN_IVF_FLAT_is_compacted_not_compacted_segment_status_all_is_string_indexed_is_string_indexed_replica_number_1_is_deleted_is_deleted_data_size_3000 (common_func.py:1393)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:15:45 - DEBUG - ci_test]: Dynamic field is enabled: False (common_func.py:1395)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:15:46 - DEBUG - ci_test]: (api_request)  : [Collection.insert] args: [      int64   float varchar                                      binary_vector

[2024-01-21T10:18:11.056Z] 0         0     0.0       0  b'\x1d\x89\xfaK3\x8c\xa7{\xbf\xee\x01t\xbdw\x12-'

[2024-01-21T10:18:11.056Z] 1         1     1.0       1                   b'()Wz\x06A-.S\xca\t\x88\x80kl='

[2024-01-21T10:18:11.056Z] 2         2     2.0       2  b'\x07\xca\xc0\xa9\x02U\x92\xfdi\......, kwargs: {'timeout': 180} (api_request.py:62)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:15:46 - DEBUG - ci_test]: (api_response) : (insert count: 3000, delete count: 0, upsert count: 0, timestamp: 447173662158618629, success count: 3000, err count: 0)  (api_request.py:37)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:15:46 - DEBUG - ci_test]: (api_request)  : [Collection.release] args: [], kwargs: {'timeout': 180} (api_request.py:62)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:15:47 - DEBUG - ci_test]: (api_response) : None  (api_request.py:37)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:15:47 - DEBUG - ci_test]: (api_request)  : [Collection.load] args: [None, 1, 120], kwargs: {} (api_request.py:62)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:17:47 - ERROR - pymilvus.decorators]: RPC error: [wait_for_loading_collection], <MilvusException: (code=1, message=wait for loading collection timeout, collection: deploy_test_index_type_BIN_IVF_FLAT_is_compacted_not_compacted_segment_status_all_is_string_indexed_is_string_indexed_replica_number_1_is_deleted_is_deleted_data_size_3000)>, <Time:{'RPC start': '2024-01-21 10:15:47.158276', 'RPC error': '2024-01-21 10:17:47.367323'}> (decorators.py:134)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:17:47 - WARNING - pymilvus.decorators]: Retry timeout: 120s (decorators.py:100)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:17:47 - ERROR - pymilvus.decorators]: RPC error: [load_collection], <MilvusException: (code=1, message=Retry timeout: 120s, message=wait for loading collection timeout, collection: deploy_test_index_type_BIN_IVF_FLAT_is_compacted_not_compacted_segment_status_all_is_string_indexed_is_string_indexed_replica_number_1_is_deleted_is_deleted_data_size_3000)>, <Time:{'RPC start': '2024-01-21 10:15:47.144369', 'RPC error': '2024-01-21 10:17:47.367741'}> (decorators.py:134)

[2024-01-21T10:18:11.056Z] [2024-01-21 10:17:47 - ERROR - ci_test]: Traceback (most recent call last):

[2024-01-21T10:18:11.056Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 32, in inner_wrapper

[2024-01-21T10:18:11.056Z]     res = func(*args, **_kwargs)

[2024-01-21T10:18:11.056Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 63, in api_request

[2024-01-21T10:18:11.056Z]     return func(*arg, **kwargs)

[2024-01-21T10:18:11.056Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/orm/collection.py", line 419, in load

[2024-01-21T10:18:11.056Z]     conn.load_collection(

[2024-01-21T10:18:11.056Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 135, in handler

[2024-01-21T10:18:11.056Z]     raise e from e

[2024-01-21T10:18:11.056Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 131, in handler

[2024-01-21T10:18:11.056Z]     return func(*args, **kwargs)

[2024-01-21T10:18:11.056Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 170, in handler

[2024-01-21T10:18:11.056Z]     return func(self, *args, **kwargs)

[2024-01-21T10:18:11.056Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 101, in handler

[2024-01-21T10:18:11.056Z]     raise MilvusException(

[2024-01-21T10:18:11.056Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=Retry timeout: 120s, message=wait for loading collection timeout, collection: deploy_test_index_type_BIN_IVF_FLAT_is_compacted_not_compacted_segment_status_all_is_string_indexed_is_string_indexed_replica_number_1_is_deleted_is_deleted_data_size_3000)>
```

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_kafka_cron/detail/deploy_test_kafka_cron/1750/pipeline
log:
[artifacts-kafka-cluster-reinstall-1750-server-second-deployment-logs.tar.gz](https://github.com/milvus-io/milvus/files/14004070/artifacts-kafka-cluster-reinstall-1750-server-second-deployment-logs.tar.gz)

collection name: BIN_IVF_FLAT-all-is_string_indexed-is_deleted-not_compacted-1

```
[2024-01-21T10:27:04.938Z] + kubectl get pods -o wide

[2024-01-21T10:27:04.939Z] + grep kafka-cluster-reinstall-1750

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-0                                    2/2     Running                  0                18m     10.104.29.149   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-1                                    2/2     Running                  0                18m     10.104.32.199   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-2                                    2/2     Running                  1 (17m ago)      18m     10.104.25.15    4am-node30   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-etcd-0                               1/1     Running                  0                18m     10.104.29.150   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-etcd-1                               1/1     Running                  0                18m     10.104.32.200   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-etcd-2                               1/1     Running                  0                18m     10.104.25.19    4am-node30   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-exporter-776ddbc5d8-scvbw            1/1     Running                  4 (17m ago)      18m     10.104.4.70     4am-node11   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-datacoord-749c8568dc-dn9vn    1/1     Running                  0                18m     10.104.29.142   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-datanode-6d65c9dd6b-969zv     1/1     Running                  0                18m     10.104.14.106   4am-node18   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-datanode-6d65c9dd6b-q7jbp     1/1     Running                  0                18m     10.104.32.190   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-indexcoord-545d69978-tdrng    1/1     Running                  0                18m     10.104.32.191   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-indexnode-7ffcf5ccc7-26ff6    1/1     Running                  0                18m     10.104.14.105   4am-node18   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-indexnode-7ffcf5ccc7-fdrfm    1/1     Running                  0                18m     10.104.29.138   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-indexnode-7ffcf5ccc7-mpvdj    1/1     Running                  0                18m     10.104.32.188   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-proxy-85d4bc6c54-mbdr7        1/1     Running                  0                18m     10.104.32.189   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-querycoord-c6dfd8d4f-bvxbv    1/1     Running                  0                18m     10.104.29.137   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-querynode-6dfdcc7779-cts5l    1/1     Running                  0                18m     10.104.32.192   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-querynode-6dfdcc7779-jlpft    1/1     Running                  0                18m     10.104.29.143   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-querynode-6dfdcc7779-qqx84    1/1     Running                  0                18m     10.104.14.107   4am-node18   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-milvus-rootcoord-7d4fd86787-jxffw    1/1     Running                  1 (16m ago)      18m     10.104.29.141   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-minio-0                              1/1     Running                  0                18m     10.104.32.197   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-minio-1                              1/1     Running                  0                18m     10.104.29.155   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-minio-2                              1/1     Running                  0                18m     10.104.25.18    4am-node30   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-minio-3                              1/1     Running                  0                18m     10.104.34.203   4am-node37   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-zookeeper-0                          1/1     Running                  0                18m     10.104.32.198   4am-node39   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-zookeeper-1                          1/1     Running                  0                18m     10.104.29.148   4am-node35   <none>           <none>

[2024-01-21T10:27:05.194Z] kafka-cluster-reinstall-1750-zookeeper-2                          1/1     Running                  0                18m     10.104.34.201   4am-node37   <none>           <none>
```

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_cron/detail/deploy_test_cron/1831/pipeline
log:

[artifacts-pulsar-cluster-reinstall-1831-server-logs.tar.gz](https://github.com/milvus-io/milvus/files/14004097/artifacts-pulsar-cluster-reinstall-1831-server-logs.tar.gz)

collections:
```
deploy_test_index_type_HNSW_is_compacted_not_compacted_segment_status_all_is_string_indexed_not_string_indexed_replica_number_2_is_deleted_is_deleted_data_size_3000
deploy_test_index_type_BIN_IVF_FLAT_is_compacted_is_compacted_segment_status_all_is_string_indexed_not_string_indexed_replica_number_1_is_deleted_is_deleted_data_size_3000
deploy_test_index_type_BIN_IVF_FLAT_is_compacted_not_compacted_segment_status_all_is_string_indexed_not_string_indexed_replica_number_2_is_deleted_is_deleted_data_size_3000
```

pod info:
```
[2024-01-21T09:58:06.635Z] + kubectl get pods -o wide

[2024-01-21T09:58:06.637Z] + grep pulsar-cluster-reinstall-1831

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-bookie-0                            1/1     Running                  0                29m     10.104.29.110   4am-node35   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-bookie-1                            1/1     Running                  0                29m     10.104.25.4     4am-node30   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-bookie-init-9m49r                   0/1     Completed                0                29m     10.104.13.203   4am-node16   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-broker-0                            1/1     Running                  0                29m     10.104.13.200   4am-node16   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-etcd-0                              1/1     Running                  0                29m     10.104.29.109   4am-node35   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-etcd-1                              1/1     Running                  0                29m     10.104.34.185   4am-node37   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-etcd-2                              1/1     Running                  0                29m     10.104.25.254   4am-node30   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-datacoord-d4c97575c-gzcb2    1/1     Running                  0                29m     10.104.13.193   4am-node16   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-datanode-c5d574b64-7gkfd     1/1     Running                  0                29m     10.104.14.98    4am-node18   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-datanode-c5d574b64-7vzdg     1/1     Running                  0                29m     10.104.13.198   4am-node16   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-indexcoord-5b6c58b9f8ghknm   1/1     Running                  0                29m     10.104.14.100   4am-node18   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-indexnode-5cb447b6f-57mcr    1/1     Running                  0                29m     10.104.29.101   4am-node35   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-indexnode-5cb447b6f-jpshp    1/1     Running                  0                29m     10.104.4.60     4am-node11   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-indexnode-5cb447b6f-txzjz    1/1     Running                  0                29m     10.104.13.196   4am-node16   <none>           <none>

[2024-01-21T09:58:06.891Z] pulsar-cluster-reinstall-1831-milvus-proxy-66d95498d6-x57s5       1/1     Running                  0                29m     10.104.4.58     4am-node11   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-milvus-querycoord-578fdcbbf-55gv4   1/1     Running                  0                29m     10.104.13.197   4am-node16   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-milvus-querynode-f784ff579-d9267    1/1     Running                  0                29m     10.104.14.101   4am-node18   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-milvus-querynode-f784ff579-f845s    1/1     Running                  0                29m     10.104.4.59     4am-node11   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-milvus-querynode-f784ff579-q997n    1/1     Running                  0                29m     10.104.13.204   4am-node16   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-milvus-rootcoord-84dd867db9-7xnbb   1/1     Running                  0                29m     10.104.13.199   4am-node16   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-minio-0                             1/1     Running                  0                29m     10.104.29.108   4am-node35   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-minio-1                             1/1     Running                  0                29m     10.104.25.253   4am-node30   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-minio-2                             1/1     Running                  0                29m     10.104.34.187   4am-node37   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-minio-3                             1/1     Running                  0                29m     10.104.32.171   4am-node39   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-proxy-0                             1/1     Running                  0                29m     10.104.13.201   4am-node16   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-pulsar-init-x5s2b                   0/1     Completed                0                29m     10.104.13.202   4am-node16   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-recovery-0                          1/1     Running                  0                29m     10.104.14.99    4am-node18   <none>           <none>

[2024-01-21T09:58:06.892Z] pulsar-cluster-reinstall-1831-zookeeper-0                         1/1     Running                  0                29m     10.104.29.111   4am-node35   <none>           <none>
```

**Comment by zhuwenxing** [comment]
it is a stable reproduced issue!
failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_cron/detail/deploy_test_cron/1843/pipeline
log:
[artifacts-pulsar-cluster-reinstall-1843-server-logs.tar.gz](https://github.com/milvus-io/milvus/files/14046113/artifacts-pulsar-cluster-reinstall-1843-server-logs.tar.gz)

```
[2024-01-24T09:40:57.378Z] [2024-01-24 09:38:22 - DEBUG - ci_test]: (api_request)  : [Collection.load] args: [None, 2, 120], kwargs: {} (api_request.py:62)

[2024-01-24T09:40:57.378Z] [2024-01-24 09:40:23 - ERROR - pymilvus.decorators]: RPC error: [wait_for_loading_collection], <MilvusException: (code=1, message=wait for loading collection timeout, collection: deploy_test_index_type_BIN_IVF_FLAT_is_compacted_is_compacted_segment_status_all_is_string_indexed_not_string_indexed_replica_number_2_is_deleted_is_deleted_data_size_3000)>, <Time:{'RPC start': '2024-01-24 09:38:22.963921', 'RPC error': '2024-01-24 09:40:23.132140'}> (decorators.py:134)

[2024-01-24T09:40:57.378Z] [2024-01-24 09:40:23 - WARNING - pymilvus.decorators]: Retry timeout: 120s (decorators.py:100)

[2024-01-24T09:40:57.378Z] [2024-01-24 09:40:23 - ERROR - pymilvus.decorators]: RPC error: [load_collection], <MilvusException: (code=1, message=Retry timeout: 120s, message=wait for loading collection timeout, collection: deploy_test_index_type_BIN_IVF_FLAT_is_compacted_is_compacted_segment_status_all_is_string_indexed_not_string_indexed_replica_number_2_is_deleted_is_deleted_data_size_3000)>, <Time:{'RPC start': '2024-01-24 09:38:22.942586', 'RPC error': '2024-01-24 09:40:23.132416'}> (decorators.py:134)

[2024-01-24T09:40:57.378Z] [2024-01-24 09:40:23 - ERROR - ci_test]: Traceback (most recent call last):

[2024-01-24T09:40:57.378Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 32, in inner_wrapper

[2024-01-24T09:40:57.378Z]     res = func(*args, **_kwargs)

[2024-01-24T09:40:57.378Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 63, in api_request

[2024-01-24T09:40:57.378Z]     return func(*arg, **kwargs)

[2024-01-24T09:40:57.378Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/orm/collection.py", line 419, in load

[2024-01-24T09:40:57.378Z]     conn.load_collection(

[2024-01-24T09:40:57.378Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 135, in handler

[2024-01-24T09:40:57.378Z]     raise e from e

[2024-01-24T09:40:57.378Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 131, in handler

[2024-01-24T09:40:57.378Z]     return func(*args, **kwargs)

[2024-01-24T09:40:57.378Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 170, in handler

[2024-01-24T09:40:57.378Z]     return func(self, *args, **kwargs)

[2024-01-24T09:40:57.378Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 101, in handler

[2024-01-24T09:40:57.378Z]     raise MilvusException(

[2024-01-24T09:40:57.378Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=Retry timeout: 120s, message=wait for loading collection timeout, collection: deploy_test_index_type_BIN_IVF_FLAT_is_compacted_is_compacted_segment_status_all_is_string_indexed_not_string_indexed_replica_number_2_is_deleted_is_deleted_data_size_3000)>

[2024-01-24T09:40:57.378Z]  (api_request.py:45)
```

**Comment by aoiasd** [comment]
If a loading segment has been compacted and dropped, and gc delete files, will cause load failed with NOSUCHKEY.

Because we will update target if load collection failed every 5 mins, and load task's timeout limit was 10 mins, we only have once chance to update target and reload before load timeout.

**Comment by ThreadDao** [comment]
image: master-20240130-fda6f8d0-amd64
[deploy_test_kafka_cron/1791/](https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_kafka_cron/detail/deploy_test_kafka_cron/1791/pipeline/153)

**Comment by aoiasd** [comment]
May relate https://github.com/milvus-io/milvus/issues/30366, seems not delete by gc (no gc logs),  but segment load with wrong logID.

**Comment by aoiasd** [comment]
Log path compress to log ID has conflict with some old codes

**Comment by zhuwenxing** [comment]
it reproduced stably

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_kafka_cron/detail/deploy_test_kafka_cron/1867/pipeline

**Comment by xiaofan-luan** [comment]
/assign @wangting0128 
can you verify again?

**Comment by wangting0128** [comment]
> /assign @wangting0128 can you verify again?

@zhuwenxing please help to verify it

**Comment by zhuwenxing** [comment]
verified and fixed in `master-20240307-48637655`
